code/datums/components/crafting/recipes/recipes_misc.dm

```diff
@@ -495,6 +488,29 @@
 	category = CAT_MISCELLANEOUS
 */
```

Comment:
throw out the remaining comments above, please?

Reply:
I meant all of them...

Reply:
I like leaving in things that haven't been added yet, so we have a reference to work off of if we ever add them. The extendo hand, for instance. I think there's value in retaining it as a concept/recipe for the future. I often benefit from these kinds of notes.